### PR TITLE
Add cross-compiled DMG archives

### PR DESCRIPTION
**Pull request by heinrich5991**
Currently they only contain the client and the archives are a lot larger
than the current release artifacts.

### CODE REVIEW

**Comment by heinrich5991** [comment]
Requires #940. Don't merge yet, it's still ugly and contains at least one TODO.

What do you think?

**Comment by def-** [comment]
Would be great to have this. I can't figure out how to update clang on my mac vm, so right now i can't compile ddnet anymore. How to get OSX SDK on Arch Linux? I guess you have to buy Mac OSX first, right?

**Comment by heinrich5991** [comment]
@def- You can follow the instructions I put into the README, it should be in the manual of the linked [osxcross](https://github.com/tpoechtrager/osxcross), under ["Packaging the SDK on Linux, Method 1"](https://github.com/tpoechtrager/osxcross/tree/1a1733a773fe26e7b6c93b16fbf9341f22fac831#packing-the-sdk-on-linux-method-1-works-up-to-xcode-73).

File: src/osxlaunch/server.mm [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		defer: NO];
+
+	[window setTitle: @"DDNet Server"];
+
+	view = [[[ServerView alloc] initWithFrame: graphicsRect] autorelease];
+	[view setEditable: NO];
+	[view setRulerVisible: YES];
+
+	[window setContentView: view];
+	[window setDelegate: (id<NSWindowDelegate>)view];
```
Comment (heinrich5991) [inline src/osxlaunch/server.mm]: I added the cast to `(id<NSWindowDelegate>)` here, but I'm not sure if it is correct, as I don't understand the Objective-C++ here.

**Comment by heinrich5991** [comment]
@def- It should be good to merge now, I tested it in a macOS VM.

**Pull request merged**